src/ImageProcessor/Colors/ColorSpacialTransforms.cs

```diff
@@ -0,0 +1,26 @@
+﻿// <copyright file="ColorSpacialTransforms.cs" company="James Jackson-South">
+// Copyright (c) James Jackson-South and contributors.
+// Licensed under the Apache License, Version 2.0.
+// </copyright>
+
+namespace ImageProcessor
+{
+    public struct ColorSpacialTransforms
```

Comment:
Tell me about it! I'm banging my head against a brick wall just now trying to speed up the resampler.